cargo-apk/src/apk.rs

```diff
@@ -84,7 +84,7 @@ impl<'a> ApkBuilder<'a> {
             if self.cmd.target().is_none() {
                 cargo.arg("--target").arg(triple);
             }
-            cargo.args(self.cmd.args());
+            self.cmd.args().apply(&mut cargo);
```

Comment:
The `Args` struct in `cargo-subcommand` only holds and applies a limited set of args now, how about (new) arguments that are not (yet) recognized by it? Ie. all flags related to features are not in the struct and hence won't be forwarded to `cargo`.

Perhaps we should keep storing and passing the original set of arguments as-is?